DBlock Game/DBlock Game/basement.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DBlock_Game
+{
+    internal class basement
```

Comment:
its probably not worth the effort at this point to add extra classes

Reply:
yea i tried it out and wasnt going to work out and forgot to delete